server/routes/functionalSkills/functionalSkillsController.test.ts

```diff
@@ -71,7 +66,7 @@ describe('functionalSkillsController', () => {
       const prisonerSummary = aValidPrisonerSummary(prisonNumber)
       req.session.prisonerSummary = prisonerSummary
 
-      prisonService.lookupPrison.mockImplementation(mockPrisonLookup)
+      prisonService.getPrisonByPrisonId.mockImplementation(mockPrisonLookup)
```

Comment:
The method exposed by `prisonService` and used by all dependant code is now `getPrisonByPrisonId` rather than `lookupPrison` 👍